pyglet/event.py

```diff
@@ -168,6 +168,17 @@ class EventException(Exception):
     pass
 
 
+def register_event_type(name):
```

Comment:
It might be a good idea to add a check to make sure the decorated class is a subclass of `pyglet.EvenDispatcher`: 
```py
if not issubclass(cls, EventDispatcher):
    raise NotSureTheBestException("Events types can only be registered on EventDispatcher subclasses")
```

Reply:
added type check